contributing-frontend.md

```diff
@@ -72,39 +78,99 @@ const { t, ready } = i18nReact.useTranslation(undefined, { useSuspense: false })
 
 The useTranslation hook is automatically mocked in the unit tests. The json based language files used by i18nReact are located in folder [src/desktop/static/desktop/locales](https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/static/desktop/locales/) and the default namespace is `translation`. See [i18next](https://github.com/i18next/i18next) for more info.
 
-
-
 ### Application wide communication
+
 For communication with non react based parts of Hue there is a publish–subscribe messaging system called [huePubsub](https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/js/utils/huePubSub.ts). To publish a message, call the `publish` function with a topic. You can subscribe to a topic using the hook [usePubSub](https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/js/reactComponents/useHuePubSub.ts) which will force your functional component to rerender once a message matching the topic is published. In the example below the editCursor state is updated by useHuePubSub each time a CURSOR_POSITION_CHANGED_EVENT is published.
 
 ```js
-const editorCursor = useHuePubSub<EditorCursor>({ topic: CURSOR_POSITION_CHANGED_EVENT });
+const editorCursor = useHuePubSub < EditorCursor > { topic: CURSOR_POSITION_CHANGED_EVENT };
```

Comment:
Nit: Doesn't seem right, the original was good no?

Reply:
You are correct. It was Prettier that was being clever and modifying the code example. Changed the code example language from js to ts.